omod/src/main/java/org/openmrs/module/appointments/web/mapper/AppointmentMapper.java

```diff
@@ -252,6 +250,9 @@ private Map createPatientMap(Patient p) {
         map.put("name", p.getPersonName().getFullName());
         map.put("uuid", p.getUuid());
         map.put("identifier", p.getPatientIdentifier().getIdentifier());
+        map.put("age", p.getAge());
+        map.put("gender", p.getGender());
+        map.putAll(p.getActiveIdentifiers().stream().filter(e -> e.getIdentifierType() != null).collect(Collectors.toMap(e -> e.getIdentifierType().toString().replaceAll("[- ]", ""), e -> e.getIdentifier())));
```

Comment:
What does it put in the map? 
From the code - seems the key is the "identifier type" and value is the identifier. 
Also from the replaceall expression, this seems very specific to your requirements. 

Also note, you are using "PatientIdentifierType.toString()" - which is just sending back "identifier" value - this will result in something like "HID101" : "HID101". I am not sure I understand the requirement. 

This should not be so, if you want all identifiers, then I prefer you sending as a top level attribute to the patient map object .. maybe call "identifiers" (as we are already sending the primary identifier through patient.identifier). 
If you do not require that in the map, kindly remove this line.